dbt/runner.py

```diff
@@ -500,6 +527,20 @@ def safe_execute_node(self, data):
                     node,
                     error='{}\n'.format(ABORTED_TRANSACTION_STRING),
                     status="SKIP")
+        except dbt.exceptions.InternalException as e:
+            error = ("Internal error executing {filepath}\n\n{error}"
+                     "\n\nThis is an error in dbt. Please try again. If "
+                     "the error persists, open an issue at "
+                     "https://github.com/fishtown-analytics/dbt").format(
+                         filepath=node.get('build_path'),
+                         error=str(e).strip())
+            status = "ERROR"
+            if type(e) == psycopg2.InternalError and \
+               ABORTED_TRANSACTION_STRING == e.diag.message_primary:
```

Comment:
i think we can leave it in for now. but if overall error handling has improved, i don't think this particular error message is incredibly helpful (since each model gets its own transaction)